Source/DafnyCore/Options/InMemoryDirectoryInfoFromDotNet8.cs

```diff
@@ -0,0 +1,139 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.FileSystemGlobbing.Abstractions;
+
+namespace DafnyCore.Options;
+
+/// <summary>
+/// Contains a bugfix that's available in DotNet8
```

Comment:
Could we just require .NET 8 now instead of copying this code?

Reply:
It's in preview